src/libstore/build.cc

```diff
@@ -2499,13 +2503,20 @@ void DerivationGoal::runChild()
             }
             sandboxProfile += ")\n";
 
+            sandboxProfile += additionalSandboxProfile;
+
             debug("Generated sandbox profile:");
             debug(sandboxProfile);
 
+            char *tmpProfile = strdup((format("%1%/nix-sandboxXXXXXX.sb") % globalTmpDir).str().c_str());
+            int profileFd = mkstemps(tmpProfile, 3);
+            closeOnExec(profileFd);
+            writeFull(profileFd, sandboxProfile);
```

Comment:
Of course we can import, but how does that remove the need for 2487-2491 and 2500-2504?